events/guildDelete.js

```diff
@@ -1,14 +1,15 @@
 // This event executes when the bot leaves a guild (server).
 
 module.exports = (client, guild) => {
-   if (!guild.available) return; // If there is an outage, return.
+  if (!guild.available) return; // If there is an outage, return.
   // Update the activity to show the updated number of guilds.
-  client.user.setActivity(`${client.settings.get("default").prefix}help | ${client.guilds.cache.size} Servers`);
+  client.user.setActivity(`${client.getSettings("default").prefix}help | ${client.guilds.cache.size} Servers`, {type: "PLAYING"});
```

Comment:
Same here, not sure why you had to add the playing type as that is the default.

Reply:
I didn't knew it was default